src/igvfd/schemas/changelogs/primary_cell.md

```diff
@@ -1,6 +1,11 @@
 ## Changelog for *`primary_cell.json'*
 
+### Schema version 7
```

Comment:
Not sure the version should be 7, should not it be 6?
I think better would be "Require 'biosample_term'."